org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -45,6 +70,37 @@
         }
       }
     }
+
+    @media (max-width: 1340px) {
```

Comment:
Yes they work, I tried with `max-device-width` and didn't work. The tests were made on Chrome and Firefox.